### PR TITLE
[Feature]: Support Unit5 Test For Clickhouse Connector

### LINKED ISSUE #138: [Feature]: Support Unit5 Test For  Clickhouse Connector
### What happened?

There is currently a lack of basic unit testing facilities, which results in many problems not being discovered in time. We should refer to other connectors in flink to build complete unit tests. At present, it seems that using **testcontainer & flink mini cluster** testing is a common practice

### Affects Versions

master

### What are you seeing the problem on?

Ci

### How to reproduce

_No response_

### Relevant log output

_No response_

### Anything else

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by qingfei1994**
## Why are the changes needed?

Close #138.

## Brief change log

- End2End test case for clickhouse connector

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (no)

### CODE REVIEW

File: flink-connector-clickhouse-e2e-test/pom.xml [hunk 1]
```diff
@@ -35,6 +35,141 @@ limitations under the License.
...
 	<dependencies>
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-connector-test-utils</artifactId>
+			<version>${flink.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.testcontainers</groupId>
+			<artifactId>clickhouse</artifactId>
+			<version>1.19.8</version>
```
Comment (czy006) [inline flink-connector-clickhouse-e2e-test/pom.xml:46]: maybe can use ${testcontainer.version} replace it

**Review by czy006 — COMMENTED** [review]
Thanks for your contribution, CI trigger didn't seem to pass I left some comments

**Comment by qingfei1994** [comment]
Let me fix the ci.yaml then.

> image
> 
> It seems to lost docker env,maybe ci add it? cc @qingfei1994

File: flink-connector-clickhouse-e2e-test/pom.xml (hunk 1 above)
Comment (qingfei1994) [reply flink-connector-clickhouse-e2e-test/pom.xml]: I've fixed it, please check

**Review by czy006 — APPROVED** [review]
LGTM

**Review by czy006 — APPROVED** [review]
